src/Microsoft.Dnx.Tooling/ClearCache/ClearCacheCommand.cs

```
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal class ClearCacheCommand
+    {
+        public Reports Reports { get; }
+        public string HttpCacheDirectory { get; }
+        public ClearCacheCommand(Reports reports, string httpCacheDirectory)
+        {
+            Reports = reports;
+            HttpCacheDirectory = httpCacheDirectory;
+        }
+        
+        public int Execute()
+        {
+            if(Directory.Exists(HttpCacheDirectory))
+            {
+                Reports.Information.WriteLine($"Clearing cache directory {HttpCacheDirectory}");
+
+                Exception exception = null;
+
+                for (var i = 0; i < 3; i++)
+                {
+                    try
+                    {
+                        DeleteDirectoryRecursively(HttpCacheDirectory, false);
+                        Reports.Information.WriteLine("Cache cleared.");
+                        return 0;
+                    }
+                    catch (Exception e)
+                    {
+                        exception = e;
```

We silently ignore the first failures? Shouldn't this *at least* warn about any errors encountered?